### DISCUSSION TITLE
GoproHero10 control using C# with USB

### DISCUSSION THREAD

**Discussion by disneyono** [gopro/OpenGoPro#276] [Q&A] [answered]
Hello.
I want to control GoproHero10 using C# with USB.
Does anyone have a sample code, etc.?
The main control I want to do is to turn the shutter on/off.

**Comment by tcamise-gpsw** [comment, marked as answer]
Hello. Yes, see the webcam demo here: https://gopro.github.io/OpenGoPro/demos/csharp/webcam

**Comment by disneyono** [comment]
Thanks for replying.
Can this web camera program record videos?
If not, is there any sample code to record a video?
If so, I would appreciate it if you could tell me.

**Comment by tcamise-gpsw** [comment]
No, this demo does not record videos. Are you referring to recording the videos on the camera's SD card or on your PC?
